packages/cli/src/commands/generate/helpers.ts

```diff
@@ -5,9 +5,10 @@ import pluralize from 'pluralize'
 import Listr from 'listr'
 import pascalcase from 'pascalcase'
 import { paramCase } from 'param-case'
-
+import { CommandModule } from 'yargs'
 import { generateTemplate, getPaths, writeFilesTask } from 'src/lib'
 import c from 'src/lib/colors'
+import { Paths } from '@redwoodjs/internal/src/types'
```

Comment:
If you run `yarn build` in the RedwoodJS root this should be available via `@redwoodjs/internal`

Reply:
Seems like only paths and config get exported, `import { Paths } from '@redwoodjs/internal` doesn't work 🧐 

```ts
// internal/src/main.ts
export * from './paths'
export * from './config'

```

Reply:
I think we need something like this: https://github.com/redwoodjs/redwood/blob/c8cf139c6fe2881801613d1f0e0bce38b55322b7/packages/core/tsconfig.json#L9-L10

Reply:
That didn't work unfortunately (I think because ./types isn't exported).
I (re-)exported the Paths type from ./paths, let me know if that solution sucks.

Reply:
Ah damn, you're right, that's fine 👍